devito/compiler.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, *args, **kwargs):
         super(GNUCompiler, self).__init__(*args, **kwargs)
         self.cc = 'g++'
         self.ld = 'g++'
-        self.cflags = ['-O3', '-g', '-fPIC', '-Wall']
+        self.cflags = ['-O3', '-fPIC', '-Wall']
```

Comment:
No, by "debug leftover" I meant the commented out line in the new KNL compiler settings. We should only have real comments in the main code, never commented out code, unless we have a very good reason.